pyproject.toml

```diff
@@ -30,10 +30,10 @@ dependencies = [
 
 [project.optional-dependencies]
 stan = ["bridgestan >= 2.1.2"]
-pymc = ["pymc >= 5.5.0", "numba >= 0.57.1"]
+pymc = ["pymc >= 5.8.0", "numba >= 0.57.1"]
 all = [
     "bridgestan >= 2.1.2",
-    "pymc >= 5.5.0",
+    "pymc >= 5.8.0",
```

Comment:
This isn't needed. The new path is the correct one for the older versions as well

Reply:
reverted any changes to pymc versions